lib/scooter_service.dart

```diff
@@ -44,10 +44,10 @@ class ScooterService {
 
   Future<List<BluetoothDevice>> getSystemScooters() async {
     // See if the phone is already connected to a scooter. If so, hook into that.
-    List<BluetoothDevice> systemDevices = await FlutterBluePlus.systemDevices;
+    List<BluetoothDevice> systemDevices = await FlutterBluePlus.bondedDevices;
```

Comment:
Could you check if that would also work for iOS devices?

Of course that would require that the devices is already bonded. The nRF connect app also provides a list of all bonded devices. Therefore it sounds right to me.

Eventually we could also search in both lists. Wdyt?

Reply:
Great, comment! I checked it again and just need some legacy permissions for my legacy android phone.